starter-code/index.html

```diff
@@ -0,0 +1,112 @@
+                    <button class="btn rectangle-btn">Read more</button>
+                </div>
+            </div>
+            <div class="carousel-item">
+                <img class="d-block w-100" src="img/project-2.jpg" alt="Second slide">
+                <div class="carousel-caption">
+                    <h1>Lorem ipsum dolor sit amet, consectetur adipiscing elit</h1>
+                    <p>Vestibulum faucibus sagittis ante ullamcorper dignissim. Aliquam tincidunt convallis diam congue malesuada. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi et ligula vitae ligula imperdiet sodales vel eu diam. Aliquam erat volutpat. Curabitur sit amet vehicula mi. Nulla volutpat tincidunt erat id tincidunt. Sed maximus pretium feugiat. Nulla tincidunt nunc in turpis iaculis, non consectetur nulla pretium. Nulla in venenatis quam. Etiam at pharetra est, a venenatis lacus. Nulla blandit lorem vitae mauris porttitor, eget posuere erat facilisis. Aliquam facilisis dui vitae elit euismod vehicula. Proin nec sodales tortor.</p>
+                    <button class="btn rectangle-btn">Read more</button>
+                </div>
+            </div>
+            <div class="carousel-item">
+                <img class="d-block w-100" src="img/project-3.jpg" alt="Second slide">
+                <div class="carousel-caption">
+                    <h1>Lorem ipsum dolor sit amet, consectetur adipiscing elit</h1>
+                    <p>Vestibulum faucibus sagittis ante ullamcorper dignissim. Aliquam tincidunt convallis diam congue malesuada. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi et ligula vitae ligula imperdiet sodales vel eu diam. Aliquam erat volutpat. Curabitur sit amet vehicula mi. Nulla volutpat tincidunt erat id tincidunt. Sed maximus pretium feugiat. Nulla tincidunt nunc in turpis iaculis, non consectetur nulla pretium. Nulla in venenatis quam. Etiam at pharetra est, a venenatis lacus. Nulla blandit lorem vitae mauris porttitor, eget posuere erat facilisis. Aliquam facilisis dui vitae elit euismod vehicula. Proin nec sodales tortor.</p>
+                    <button class="btn rectangle-btn">Read more</button>
+                </div>
+            </div>
+        </div>
+        <a class="carousel-control-prev" href="#carouselControls" role="button" data-slide="prev">
+            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+            <span class="sr-only">Previous</span>
+        </a>
+        <a class="carousel-control-next" href="#carouselControls" role="button" data-slide="next">
+            <span class="carousel-control-next-icon" aria-hidden="true"></span>
+            <span class="sr-only">Next</span>
+        </a>
+    </div>
+    <!-- what they do -->
+    <div class="row red-background">
```

Comment:
You can use more things apart divs, like sections and articles. Remember: semantic HTML.